README.md

```diff
@@ -55,7 +55,7 @@ Start an anger instance against a given target.
     * `connections`: The number of connections to maintain to server. Each is an individual nes client.
     * `senders`: The number of connections that should be senders.
     * `responses`: The number of overall expected responses to be recieved during this run.
-    * `auth`: An `Object` passed to the nes client for authentication. [reference.][nes-auth]
+    * `auth`: A `Function` passed to the nes client for authentication. This is passed the `client` and `index` of that client as params. Must return the auth object options. [reference.][nes-auth]
```

Comment:
Some notes:

* I think we should support both objects and functions
* _if_ the only reason for having an `index` is to know if it is one of senders, then let's add an explicit flag into the `client`, `true` or `false` (no falsey).

@mtharrison what do you think?